.github/workflows/main.yml

```diff
@@ -51,6 +51,8 @@ jobs:
       run: |
         brew install libomp
         ls /usr/local/opt/libomp/lib/
+        export OMP_NUM_THREADS=1
```

Comment:
not sure this survives between the different run blocks. I would put this just above the pytest